docs/index.md

```diff
@@ -22,6 +24,9 @@ through bisection and version analysis.
 Further information on the infrastructure architecture is available
 [here](contributing/architecture.md).
 
+[OSV-Scanner](https://google.github.io/osv-scanner/), is the first-party tool
```

Comment:
nit: no comma after OSV-Scanner?

Reply:
Fixed in https://github.com/google/osv.dev/pull/2112/commits/baa3c85781da3411e62d5beb9955558b34bd7202